### PR TITLE
[Duplicate]Introducing bring your own key(BYOK) in your systems

### LINKED ISSUE #3840: Introducing Bring Your Own Key(BYOk) in your systems
We ask that you please be patient as our team works through approving and publishing all articles/tutorials in a timely manner. Allow 1-3 days for a topic to be reviewed and/or approved - allow 3-7 days for an articles to be reviewed and/or published.

Be sure to visit our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for tools, resources, and example articles to go over.

**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**NOTE: We tend to stray away or tend not to publish reviews/comparisons of commercial product offerings.**

*We're looking for the first 2-3 paragraphs of the article that appropriately summarize what your article will be about.*

### Proposed title of article
Introducing Bring Your Own Key(BYOk) in your systems
### Introduction paragraph (2-3 paragraphs):
Data integrity forms a crucial thing in an enterprise and incase of compromisation an organization feels like it back to start line again. Data breach has become a vital and daily talk in the current world with hackers leaking alot of data. This is all because weka encryption or even unsecured data which ends up on unsafe hands. From the launch of Bring Your Own Key technique the organizations and other premises will be in a position to generate and have authority over the entire key and sharing process. using devices which are well off and tamper resistant will be the only way to strengthen the security. These devices will work on a principle of hardware security module which will help keep track of the secure export of all keys generated. For a safer data store consider implementing Bring Your Own Key where you will be the pilot.

### Key takeaways: 

1. Data security and integrity is a crucial thing in an organization which proves the genuinity of data stored.
2. Breaching of data can result to loss of income and also lose of clients due to lack of enough trust. 
3. Bring Your Own Key is a topology which is controled from personal level and has no third party outside the organization operating to store the data.

### References: 
Ulz, T., Pieber, T., Steger, C., Haas, S., Bock, H., & Matischek, R. (2017, March). Bring your own key for the industrial Internet of Things. In 2017 IEEE International Conference on Industrial Technology (ICIT) (pp. 1430-1435). IEEE.
`https://cpl.thalesgroup.com/encryption/key-management`

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by ROYFORDWANYOIKE2001**
## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This issue fixes  #3840

### CODE REVIEW

**Comment by ahmadmardeni1** [comment]
Good afternoon and thank you for submitting your article for publishing.
Your article has been entered into our queue and should be reviewed (and published pending any changes) as soon as a peer reviewer is finished with the ones in the queue before it. :rocket:
Thank you.

**Comment by ROYFORDWANYOIKE2001** [comment]
Hello @ahmadmardeni1 @Kayere @LinusMuema @paulodhiambo am done with the PR.

**Comment by WanjaMIKE** [comment]
Jumping on this one

**Comment by lalith1403** [comment]
plagiarism check done ✔️

File: content/articles/introducing-bring-your-own-key/index.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+### Introduction
+  Information honesty shapes something essential in a venture. Thus, in giving and taking, an association feels like it is back to begin line once more. Information break has turned into an imperative and everyday talk in the current. This is in conjunction with programmers releasing a great deal of information. This is because weak encryption or even unstable information ends up in risky hands. The launch of BYOK gives premises a chance to create their keys. Moreover, they will have authority over the whole key and sharing cycle. Utilizing wealthy and alter safe gadgets will be the best way to reinforce security. These gadgets will deal with a standard of the equipment security module. This is because they will assist ensure the safety of the keys. For a more secure information store, consider carrying out Bring Your Own Key, where you will be the pilot.
```
Comment (WanjaMIKE) [inline content/articles/introducing-bring-your-own-key/index.md:5]: this paragraph is full of grammatical errors

File: content/articles/introducing-bring-your-own-key/index.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+### Introduction
+  Information honesty shapes something essential in a venture. Thus, in giving and taking, an association feels like it is back to begin line once more. Information break has turned into an imperative and everyday talk in the current. This is in conjunction with programmers releasing a great deal of information. This is because weak encryption or even unstable information ends up in risky hands. The launch of BYOK gives premises a chance to create their keys. Moreover, they will have authority over the whole key and sharing cycle. Utilizing wealthy and alter safe gadgets will be the best way to reinforce security. These gadgets will deal with a standard of the equipment security module. This is because they will assist ensure the safety of the keys. For a more secure information store, consider carrying out Bring Your Own Key, where you will be the pilot. 
+  
+ [toc]
```
Comment (WanjaMIKE) [inline content/articles/introducing-bring-your-own-key/index.md:7]: incomplete table of contents

File: content/articles/introducing-bring-your-own-key/index.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  Information honesty shapes something essential in a venture. Thus, in giving and taking, an association feels like it is back to begin line once more. Information break has turned into an imperative and everyday talk in the current. This is in conjunction with programmers releasing a great deal of information. This is because weak encryption or even unstable information ends up in risky hands. The launch of BYOK gives premises a chance to create their keys. Moreover, they will have authority over the whole key and sharing cycle. Utilizing wealthy and alter safe gadgets will be the best way to reinforce security. These gadgets will deal with a standard of the equipment security module. This is because they will assist ensure the safety of the keys. For a more secure information store, consider carrying out Bring Your Own Key, where you will be the pilot. 
+  
+ [toc] 
+  - Introduction 
+  - How is BYOK different from HYOK? 
+  - Importance of introducing BYOKS 
+  - Troubleshooting BYOK errors
+
+
+### Pre-requisites
```
Comment (WanjaMIKE) [inline content/articles/introducing-bring-your-own-key/index.md:14]: How is the conclusion a prerequisite?

File: content/articles/introducing-bring-your-own-key/index.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  - Troubleshooting BYOK errors
+
+
+### Pre-requisites 
+- Data Encryption in the Cloud
+- Troubleshooting BYOKS 
+- Conclusion 
+
+### Getting Started 
+Using good and tamper-resistant devices become the only way to strengthen security. This is because these devices will work on a principle of the hardware security module. This principle will help keep track of the secure export of all keys generated. For a safer data store, consider implementing Bring Your Own Key, where you will be the pilot. Security in most organizations has been getting weak daily as technology advances. There are many cybersecurity experts, both ethical and unethical today. They are working hard to try to break the codes. Thus the need resulted in bringing your own key mechanism.
```
Comment (WanjaMIKE) [inline content/articles/introducing-bring-your-own-key/index.md:20]: I found grammatical errors in this paragraph

**Comment by WanjaMIKE** [comment]
It seems this PR is still quite a bit aways from being completed. It also seems this PR is rather light on substance. We will be closing this and please re-submit when it is ready to be published as a new PR (and you can add the duplicate label) and ping the reviewer.

Closing a PR does NOT mean the article can not be published (or fixed) it simply means that the edits, revisions, and fixes will happen outside of the queue. 👍 Be sure to consult previously published materials to get a better understanding of the level of quality the community expects.

Please feel free to go through our provided resources as well - those are there to help all our students improve their overall technical writing. To be able to take those skills to any position or program.

**Pull request closed without merging**